v2/emailpassword/nextjs/app-directory/protecting-route.mdx

```diff
@@ -389,7 +392,7 @@ import type { JwtHeader, JwtPayload, SigningKeyCallback } from "jsonwebtoken";
 import { appInfo } from "../config/appInfo";
```

Comment:
remove this import if its not needed.

Reply:
thanks for catching it. I have removed it.